source/applicability/basic_structure.rst

```diff
@@ -1191,8 +1191,10 @@ Command messages are used to give order using one or more commands, for the
 referenced object.
 The site responds with a command acknowledgement.
 
-All arguments needs to included in a command, otherwise it results a serious
-error resulting in MessageNotAck. See section about :ref:`incomplete-commands`.
+All arguments are considered required to be included in a CommandRequest unless
```

Comment:
=> All arguments in a CommandRequest are considered required unless

Reply:
Fixed